docs/api/aea_builder.md

```diff
@@ -215,6 +215,12 @@ builder.add_component_instance(...)
 # second call
 my_aea_2 = builder.builder()
 
+<a id="aea.aea_builder.AEABuilder.DEFAULT_AGENT_ACT_PERIOD"></a>
```

Comment:
Please merge the latest main and re run the doc generator or cherry pick [c410473](https://github.com/valory-xyz/open-aea/pull/590/commits/c410473daa3481d3ad3b6aadf96ca1b9042d46c2) and [3ad6af7](https://github.com/valory-xyz/open-aea/pull/590/commits/3ad6af7304a53d843afb578a2c7449c0fc8efd4e)